worker-workflow-container/pom.xml

```diff
@@ -356,7 +357,7 @@
                                     </exec>
                                 </cmd>
                                 <healthCheck>
-                                    <cmd>curl -f http://localhost:8081/healthcheck || exit 1</cmd>
+                                    <cmd>curl -f http://localhost:8080/health-check?type=READY || exit 1</cmd>
```

Comment:
I did not change the `<wait>` section below from:

```<url>http://${docker.host.address}:${worker.adminport}</url>```

to:

```<url>http://${docker.host.address}:${worker.port}/health-check?type=READY</url>```

like I did in the other PRs because the workflow worker does not start up healthy on the develop branch:

```
127.0.0.1 - - [16/May/2024:11:09:58 +0000] "GET /healthcheck HTTP/1.1" 500 542 "-" "curl/8.0.1" 6
com.hpe.caf.api.ConfigurationException: No workflows available.
        at com.github.cafdataprocessing.workflow.WorkflowManager.getWorkflows(WorkflowManager.java:146)
        at com.github.cafdataprocessing.workflow.WorkflowManager.<init>(WorkflowManager.java:53)
        at com.github.cafdataprocessing.workflow.WorkflowWorkerFactory.createDocumentWorker(WorkflowWorkerFactory.java:43)
        at com.hpe.caf.worker.document.DocumentWorkerFactoryProvider.createDocumentWorker(DocumentWorkerFactoryProvider.java:78)
        at com.hpe.caf.worker.document.DocumentWorkerFactoryProvider.getWorkerFactory(DocumentWorkerFactoryProvider.java:49)
        at com.hpe.caf.worker.core.WorkerApplication.run(WorkerApplication.java:106)
        at com.hpe.caf.worker.core.WorkerApplication.run(WorkerApplication.java:58)
        at io.dropwizard.core.cli.EnvironmentCommand.run(EnvironmentCommand.java:66)
        at io.dropwizard.core.cli.ConfiguredCommand.run(ConfiguredCommand.java:98)
        at io.dropwizard.core.cli.Cli.run(Cli.java:78)
        at io.dropwizard.core.Application.run(Application.java:94)
        at com.hpe.caf.worker.core.WorkerApplication.main(WorkerApplication.java:72)
```

It has probably gone unnoticed because no there are no integration tests, the workflow worker container just starts and then stops.

Reply:
I think there are already tickets about improving the Workflow Worker tests.